src/driver.rs

```diff
@@ -328,6 +338,15 @@ pub fn main() {
                         .map(|s| PathBuf::from(s.trim()))
                 })
                 .or_else(|| option_env!("SYSROOT").map(PathBuf::from))
+                .or_else(|| {
+                    let home = option_env!("RUSTUP_HOME")
+                        .or(option_env!("MULTIRUST_HOME"))
```

Comment:
I don't think we use multirust anymore.

Reply:
That is for people who compile clippy with a multirust setup, I guess. I'd rather leave this in in this PR and deal with it later, since this is unrelated. This file is only modified, because the move detected this bug.